Automattic/jetpack
# package.json: fix spelling of deprecated in pnpm tasks

#### Changes proposed in this Pull Request:
Fix spelling of 'deprecated' in pnpm tasks. It is currently spelt 'deprected'.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
Run a deprecated task like `pnpm build` from the Jetpack root folder. The message should read:

<img width="467" alt="Screen Shot 2021-10-14 at 14 31 35" src="https://user-images.githubusercontent.com/17325/137235378-3322ebd8-b37b-4d0d-9b31-60fd999777cc.png">
